ipaserver/plugins/idviews.py

```diff
@@ -694,6 +694,10 @@ def resolve_anchor_to_object_name(ldap, obj_type, anchor):
 
                 if type_correct:
                     return name
+        else:
+            # Without the DCERPC bindings the sid is not resolvable, return
+            # ipaAnchorUUID
+            return anchor
```

Comment:
This would return `:SID:S-1-...` value instead of the user name.

I think it is better to do something similar to what happens in the `IPA_ANCHOR_PREFIX` branch: search for an LDAP entry of the ID override in the default trust view that has `ipaAnchorUUID` equal to the anchor value and return its `ipaOriginalUid` attribute value.